### DISCUSSION TITLE
New Word List for Alias and Subalias Creation.

### DISCUSSION THREAD

**Discussion by swapnil2905** [simple-login/app#648] [Feature Requests]
Hello Team Simple login, kindly have a look at this word list [(link)](https://github.com/turtlesoupy/this-word-does-not-exist)
Since the current alias creations and sub-creations words are not user friendly. Kindly have a look into this.

**Comment by nguyenkims** [comment]
@swapnil2905 the project looks indeed interesting! Do you know what is the cost (in terms of CPU, RAM) of generating new word using this lib?

**Comment by swapnil2905** [comment]
@nguyenkims I mean it's just word list, so free tier GPU from Paperspace or google collab should do the work, it won't require Some high computing power.
